packages/appium/lib/config.js

```diff
@@ -28,6 +28,15 @@ function getNodeVersion() {
   return /** @type {import('semver').SemVer} */ (semver.coerce(process.version));
 }
 
+/**
+ * Returns version of `npm`
+ * @returns {Promise<string>}
+ */
+async function getNpmVersion() {
+  const {stdout} = await exec('npm', ['--version']);
```

Comment:
we shouldn't do anything clever to find the version of NPM installed in this node's global node modules? i can't remember whether "npm in the path" is already a requirement.

Reply:
we need `npm` to be in the `PATH` in order to install extensions, yes?

anyway this will break on windows I think. should be `npm.cmd`

Reply:
(imo finding `npm` if it _isn't_ in the `PATH` is kind of a fool's errand)

Reply:
fixed the windows business